src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -59,15 +75,23 @@ export default function withAnnotations<P extends object>(
             }
         };
 
+        handleOnAnnotator = (annotator: Annotator): void => {
+            this.annotator = annotator;
+        };
+
         handlePreviewDestroy = (): void => {
             this.setState(defaultState);
+            this.annotator = undefined;
```

Comment:
If this is a nullable property, use `null` rather than explicitly setting `undefined`.